### PR TITLE
Transparency report 2023.

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/blog/2024-01-05-transparency2023/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+of people who experience or report incidents.
+
+In 2023, we received one Code of Conduct incident report. 
+
+### Summary of reported incident
+
+* _Description of the incident:_ 
+* _Community space:_ Slack
+* _Actions:_  We followed our protocols to investigate the incident and informed the reported and reporter of our decision and the outcome. 
+* _Observations:_
```
Comment (mpadge) [inline content/blog/2024-01-05-transparency2023/index.md:29]: Do we need the "Observations" here? Also note that i think we could and maybe should add a little more detail, in particular to clarify that our mediation was pursued until we were assured that the reporter was satisfied.

**Review by mpadge — CHANGES_REQUESTED** [review]

File: content/blog/2024-01-05-transparency2023/index.md (hunk 1 above)
Comment (yabellini) [reply content/blog/2024-01-05-transparency2023/index.md:29]: This was also a skeleton to help with the report. Please don't hesitate to change it.

File: content/blog/2024-01-05-transparency2023/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+instructions on how to make a report, and information on how reports are handled. We, the Code of Conduct Committee, 
+are responsible for receiving, investigating, deciding, enforcing and reporting on all reports of potential 
+violations of our Code. We are committed to transparency with our community while upholding the privacy 
+of people who experience or report incidents.
+
+In 2023, we received one Code of Conduct incident report. 
+
+### Summary of reported incident
+
+* _Description of the incident:_
```
Comment (yabellini) [inline content/blog/2024-01-05-transparency2023/index.md:26]:
```suggestion
* _Description of the incident:_  We received a Code of Conduct report regarding sharing a paywalled article in Slack. After review, we found that this conversation did not violate the Code of Conduct guidelines. The conversation was voluntarily deleted from Slack by its participants.
```

File: content/blog/2024-01-05-transparency2023/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+violations of our Code. We are committed to transparency with our community while upholding the privacy 
+of people who experience or report incidents.
+
+In 2023, we received one Code of Conduct incident report. 
+
+### Summary of reported incident
+
+* _Description of the incident:_ 
+* _Community space:_ Slack
+* _Actions:_  We followed our protocols to investigate the incident and informed the reported and reporter of our decision and the outcome.
```
Comment (yabellini) [inline content/blog/2024-01-05-transparency2023/index.md:28]:
```suggestion

```

File: content/blog/2024-01-05-transparency2023/index.md (hunk 1 above)
Comment (yabellini) [inline content/blog/2024-01-05-transparency2023/index.md:29]:
```suggestion

```

**Review by yabellini — COMMENTED** [review]
Changes as suggested by Kara and mark.

**Review by mpadge — APPROVED** [review]

**Pull request merged**